### DISCUSSION TITLE
What a glorious day!

### DISCUSSION THREAD

**Discussion by poirazis** [Budibase/budibase#7871] [General]
Hey @melohagan , next 5 rounds are on me! 

<img width="1113" alt="Screenshot 2022-09-21 at 2 12 16 PM" src="https://user-images.githubusercontent.com/80512170/191489768-f381dde7-444e-4bc3-8b4f-d4a4d12912d8.png">

**Comment by melohagan** [comment]
You are very welcome! 😄

**Reply by poirazis** [reply]
Let the adventure begin !
https://bb.poirazis.com/app/budiblog#/home

**Reply by joebudi** [reply]
Wow - awesome idea!

**Reply by poirazis** [reply]
thanks @joebudi ! Hope you guys will join in when we will be presenting it to the public ! It would be an honor for us, and we would like to give you guys all the thanks you deserve.
